sdk/monitor/azure-monitor-opentelemetry/tests/autoinstrumentation/test_distro.py

```diff
@@ -35,20 +35,16 @@ def test_configure(self, mock_diagnostics, azure_core_mock, attach_mock):
         self.assertEqual(
             environ,
             {
-                "OTEL_METRICS_EXPORTER": "azure_monitor_opentelemetry_exporter",
-                "OTEL_TRACES_EXPORTER": "azure_monitor_opentelemetry_exporter",
-                "OTEL_LOGS_EXPORTER": "azure_monitor_opentelemetry_exporter",
-                "OTEL_TRACES_SAMPLER": "azure_monitor_opentelemetry_sampler",
                 "OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED": "true",
                 "OTEL_EXPERIMENTAL_RESOURCE_DETECTORS": "azure_app_service",
             }
         )
 
     @patch.dict("os.environ", {
-        "OTEL_METRICS_EXPORTER": "custom_metrics_exporter",
-        "OTEL_TRACES_EXPORTER": "custom_traces_exporter",
-        "OTEL_LOGS_EXPORTER": "custom_logs_exporter",
-        "OTEL_TRACES_SAMPLER": "custom_traces_sampler",
+        # "OTEL_METRICS_EXPORTER": "custom_metrics_exporter",
```

Comment:
Comments?

Reply:
Uncommented

Reply:
Might not have pushed the changes? Seems like the lines are still commented.

Reply:
Leaving uncommented. While the env var logic is now all on the configurator, it's still valuable to know the distro does not overwrite them